csm/core/controllers/routes.py

```diff
@@ -20,7 +20,7 @@
 from .onboarding import OnboardingStateView
 from .system_config import SystemConfigListView
 from .system_config import SystemConfigView
-from .storage_capacity import StorageCapacityView
+# from .storage_capacity import StorageCapacityView
```

Comment:
If not required please remove this line.

Reply:
updated